client/src/app/layout/HeaderApp/SSOMenu.tsx

```diff
@@ -69,14 +69,9 @@ export const SSOMenu: React.FC = () => {
                       LocalStorageKey.selectedPersona
                     );
                     {
-                      keycloak
-                        .logout()
-                        .then((res) => {
-                          history.push("/");
-                        })
-                        .catch((err) => {
-                          history.push("/");
-                        });
+                      keycloak.logout().finally(() => {
```

Comment:
Might be worth adding an error in for debugging. 

```suggestion
keycloak
  .logout()
  .then((res) => {
    history.push("/");
  })
  .catch((err) => {
    console.error('Logout failed:', err);
        history.push("/");
  })
```